### DISCUSSION TITLE
Proposal to change the name of the project

### DISCUSSION THREAD

**Discussion by ruben1679** [adriangb/xpresso#18] [General]
Hello, 
This project name and the logo are directly conflicting with the project owned by Cisco [Xpresso](https://developer.cisco.com/docs/pyats/#!pyats-dashboard-xpresso).
Your logo is very close as well. Since this will confuse users of PyATS and Xpresso and both the systems are public I think you can rename your project and change the logo.

Cisco's PyATS framework and Xpresso dashboard are still live and being used by the users all around the globe for 5/6 years now.

I hope you will consider my suggestion.

thanks and wish you all the best.
Ruben

**Comment by adriangb** [comment]
Hi Ruben,

I appreciate the suggestion, but I don't see that as necessary. I googled "Xpresso" before choosing the name and I did not see anything that would be confusing/conflicting, including the product(s) you linked. In fact the first result I get is this: https://xpresso.gs.washington.edu which is another open source project with a similar logo, not a Cisco product.

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/1755071/150403314-80edb17f-f63c-49f5-8aaa-a208c3d19116.png">

I also don't really see the relationship between the projects. There's some corporate mumbo jumbo on that website, but it doesn't look like a Python web framework to me. I doubt users will confuse a Python microframework with a tool for managing Agile development and test suites or whatever XPRESSO is.

 I don't even see a logo remotely similar to ours on the page you linked.

**Reply by adriangb** [reply]
> Since this will confuse users of PyATS and Xpresso

To be fair, re-reading your comment I think _I_ may have been confused about which Xpresso you were referring to here.
I read it as "the users of Cisco's XPRESSO may be confused".
If you were referring to the users of this library, I appreciate your concern.
I still do think that it won't be a problem.

**Comment by mdear** [comment]
Hello.

To illustrate Ruben's point : 

![image](https://user-images.githubusercontent.com/6562082/150479107-9731ef03-2805-45d1-9e79-dd44f1652d76.png)

versus

![image](https://user-images.githubusercontent.com/6562082/150479168-ac533b67-d79b-4a9b-8faf-913e86dc4b56.png)

**Reply by adriangb** [reply]
Could you please clarify your intentions? Are you worried about the consumers of this library getting confused? Are you worried about Cisco's customer's getting confused?

**Reply by adriangb** [reply]
[This](https://xpresso.gs.washington.edu) is the first result in a Google search by the way:
<img width="216" alt="image" src="https://user-images.githubusercontent.com/1755071/150479614-5a1e0f74-a645-4bae-8b46-20ed4eea8495.png">

And it's not Cisco's product

**Comment by adriangb** [comment]
Well folks I changed the logo to just a coffee bean, I hope that settles any concerns you had. I am not willing to change the name.

**Reply by jgould22** [reply]
Just a drive by comment.

I think the bean looks nice and the split in it is reminiscent of FastAPIs lighting bolt which I think is nice.

**Reply by adriangb** [reply]
Thanks! I don't consider myself artistic at all so I'm glad it looks good to you
